libtransmission/tr-popcount.h

```diff
@@ -27,17 +27,11 @@
 #define TR_HAVE_BUILTIN_POPCOUNT
 #endif
 #endif
-#if defined(__x86_64__) && defined(__SSE__)
+#if defined(__x86_64__) && defined(__POPCNT__)
```

Comment:
Seems to work for me with `-m32` too. Maybe (or is there a better way?)
```suggestion
#if (defined(__x86_64__) || defined(__i386__)) && defined(__POPCNT__)
```

Reply:
Done.